### ISSUE TITLE
Adding non-mastodon accounts

### ISSUE THREAD

**Issue by trozman** [anze3db/fedidevs#98]
I tried to add non-mastodon accounts (Wordpress+AP plugin, Calckey) to the pack and they weren't found.

Is this app for Mastodon-only accounts?

**Comment by anze3db** [comment]
Yes, currently only Mastodon instances are supported. I'll try to add support for other instance types, but I'm focusing on the safety and localization issues right now so I don't think I'll be able to get to this before the end of the year. Hope you understand!

**Comment by trozman** [comment]
Thanks for the explanation! 
There is absolutely no pressure or hurry or godforbid, obligation to do it :)

**Comment by fadelkon** [comment]
I'm interested in adding peertube, hubzilla and lemmy accounts to some lists.
Is the current method of loading or adding accounts too mastodon-specific? Would something ubiquitous like webfinger help here?

**Comment by anze3db** [comment]
@fadelkon thanks for letting me know about the others. Currently I'm relying on some Mastodon specific APIs and fields that other instances don't support (noindex and discoverable), so I'll have to make a few changes to my db tables before I can make webfinger work. Yes, my plan is to add support for webfinger and hopefully that will unlock all the sites listed in this thread 🤞

**Comment by tiptoptom** [comment]
Which platforms are supported right now?
